packages/scope-manager/jest.config.js

```diff
@@ -1,8 +1,17 @@
-'use strict';
```

Comment:
nit - should we leave this in so that it's parsed in strict mode for standardisation and safety?